### ISSUE TITLE
Cannot read property installed error

### ISSUE THREAD

**Issue by alvirtuoso** [neetjn/v-localize#10]
```
Uncaught TypeError: Cannot read property 'installed' of undefined
    at Function.Vue.use (eval at <anonymous>
```

I'm using vue-cli. Steps I've done:

1. npm installed
2. import { Localize } from 'v-localize';
3. Vue.use(Localize);

What could be causing this?

**Comment by neetjn** [comment]
Stupid error on my part, can't believe I didn't catch this sooner. It has to do with my build -- amd works fine in browser but it's not exporting the module as intended for commonjs and etc.

**Comment by neetjn** [comment]
Fixed, I'll have version 1.1.2 available shortly.

**Comment by neetjn** [comment]
Should now be resolved in version 1.1.2. If you would please install the latest version and confirm this behavior.
